### DISCUSSION TITLE
Implemented Lua's CoreLib/UI/CrankIndicator

### DISCUSSION THREAD

**Discussion by kaylanx** [finnvoor/PlaydateKit#87] [Show and tell]
https://github.com/user-attachments/assets/f63a1134-1172-42cf-956d-9ce8a4562f84

Been having a play and managed to get CoreLib/UI/CrankIndicator working with Swift 😄

Code needs a big tidy up before I create a PR,, and I've got an intermittent crash, but it's here if you are interested.  Feedback welcome:  https://github.com/kaylanx/PlaydateKit/commit/426abc272b6c04d5fdda5857049c92eb2505d4f6

**Comment by finnvoor** [comment]
This looks excellent, thanks! Definitely agree it would be nice to have in a separate target for people to include as needed, might have to think about how best to structure everything.

**Reply by kaylanx** [reply]
Yeah should be able to do it I've done it before at work for iOS apps. I'm not around for the next week so will probably be a couple of weeks before I get round to tidying it up.

**Comment by finnvoor** [comment]
#94
